### PR TITLE
Private Model Cost Tracking + New Fireworks Llama 3.2 Vision Instruct

### PR DESCRIPTION
**Pull request by marlenc3**
WIP but hoping to merge in today

### CODE REVIEW

File: fern/docs/content/change-log/2024-09.mdx [hunk 1]
```diff
@@ -2,6 +2,19 @@
...
+__September 30th, 2024__
+
+Meta's most recent [Llama 3.2 Vision Instruct](https://fireworks.ai/models/fireworks/llama-v3p2-90b-vision-instruct) is now available in Vellum!
+This model excels in visual recognition, image reasoning, captioning, and answering diverse questions related to images.
+
+## Private Models Cost Tracking
+
+__September 26th, 2024__
+
+Most private models that you create through our Private Models flow should now support cost tracking!
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-09.mdx:16]: [blocking] "private models" and "cost tracking" might be familiar concepts to us, but likely aren't to most readers reading our changelog. It'd be nice to elaborate on these concepts, provide examples, adn/or link to elsewhere in our docs.

File: fern/docs/content/change-log/2024-09.mdx [hunk 2]
```diff
@@ -2,6 +2,19 @@
 title: Changelog | September, 2024
 ---
 
+## Fireworks Llama 3.2 90B Vision Instruct
+
+__September 30th, 2024__
+
+Meta's most recent [Llama 3.2 Vision Instruct](https://fireworks.ai/models/fireworks/llama-v3p2-90b-vision-instruct) is now available in Vellum!
+This model excels in visual recognition, image reasoning, captioning, and answering diverse questions related to images.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-09.mdx:10]:
```suggestion
Meta's most recent open source vision model, [Llama 3.2 Vision Instruct](https://fireworks.ai/models/fireworks/llama-v3p2-90b-vision-instruct), is now available in Vellum.
This model excels in visual recognition, image reasoning, captioning, and answering diverse questions related to images and is a great open source option if you're looking for a vision model.
```

**Review by noanflaherty — APPROVED** [review]

File: fern/docs/content/change-log/2024-09.mdx (hunk 1 above)
Comment (marlenc3) [reply fern/docs/content/change-log/2024-09.mdx:16]: Good call! I'll add in!

Comment (dvargas92495) [reply fern/docs/content/change-log/2024-09.mdx:16]: Yea I would highlight the carousel at the top of the models page and how those should have costs appear in executions going forward

File: fern/docs/content/change-log/2024-09.mdx [hunk 3]
```diff
@@ -2,6 +2,22 @@
...
+Meta's most recent open source vision model, [Llama 3.2 Vision Instruct](https://fireworks.ai/models/fireworks/llama-v3p2-90b-vision-instruct), is now available in Vellum.
+This model excels in visual recognition, image reasoning, captioning, and answering diverse questions related to images and is a great open source option if you're looking for a vision model.
+
+## Private Models Cost Tracking
+
+__September 26th, 2024__
+
+Models that are now created through the Custom Model Carosel on the [models page](https://app.vellum.ai/models) will have
+[cost tracking for prompt sandboxes](/changelog/2024/2024-08#prompt-sandbox-cost-tracking) and [cost tracking for prompt deployments](/changelog/2024/2024-09#cost-tracking-for-prompt-deployment-executions-table).
+This should enable more cost tracking features for your custom models in the future.
```
Comment (noanflaherty) [inline fern/docs/content/change-log/2024-09.mdx:18]:
```suggestion
Models that are now created through the Custom Model Carousel on the [models page](https://app.vellum.ai/models) will have
[cost tracking for prompt sandboxes](/changelog/2024/2024-08#prompt-sandbox-cost-tracking) and [cost tracking for prompt deployments](/changelog/2024/2024-09#cost-tracking-for-prompt-deployment-executions-table).
This means that you'll be able to see the dollar cost of LLM calls to model providers even for your custom models.
```

**Review by noanflaherty — APPROVED** [review]
Thanks for the revs on this! Looks good to ship after these minor nits.

File: fern/docs/content/change-log/2024-09.mdx (hunk 3 above)
Comment (marlenc3) [reply fern/docs/content/change-log/2024-09.mdx:18]: 🫡  thanks for the assist! will commit these 🙏

**Pull request merged**